src/app/oauth2/middleware.js

```diff
@@ -20,13 +20,17 @@ module.exports = {
 
```

Comment:
I can't comment on the line directly, but the header on line 14 `passport_session_id: req.session.id` should be dropped. I've raised a PR for the backend to not expect to received it. https://github.com/alphagov/di-ipv-cri-uk-passport-back/pull/190